packages/ffe-cards/less/ffe-new-card.less

```diff
@@ -0,0 +1,107 @@
+.ffe-new-card {
+    background: @ffe-white;
+    box-shadow: 0 1px 4px 0 rgba(38, 38, 38, .3);
+    display: flex;
+    flex-flow: row nowrap;
+    margin: 0 0 10px;
+    padding: 10px 20px;
+    border: 2px solid @ffe-white;
+    border-radius: 5px;
+    outline: none;
+    cursor: pointer;
+    transition: border-color 0.2s;
+    width: 100%;
+    max-width: 440px;
+    min-height: 130px;
+    text-decoration: none;
+
+    &:focus {
+        box-shadow: 0 0 0 2px @ffe-blue-focus;
+    }
+
+    &:focus,
+    &:active,
+    &:hover {
+        border-color: @ffe-blue-azure;
```

Comment:
Should all cards have a hover effect?
Perhaps not the ones that aren't clickable?

Reply:
Same again, all cards are clickable so should have a hover effect.